### ISSUE TITLE
Include examples of where EveryPolitician data is being/has been used

### ISSUE THREAD

**Issue by MyfanwyNixon** [everypolitician/everypolitician#301]
Could go in http://docs.everypolitician.org/ under 'what's in the data?' perhaps.

**Comment by davewhiteland** [comment]
No: we're not burying this in what is effectively "the docs" -- this is a key thing, so I'm proposing this to become a main focus in its own right (discussed with @tmtmtmtm at some length yesterday), possibly with new examples being added as a blog once we get rolling.

**Comment by MyfanwyNixon** [comment]
Meanwhile can someone drop me a few examples for the copy I'm writing on mysociety.org? ;)

**Comment by tmtmtmtm** [comment]
* WriteInPublic.org
* peoplesmomentum.com
* gender-balance.org
* kuvakazim.com
* rada4you.org

**Comment by MyfanwyNixon** [comment]
:+1: Thanks!

**Comment by zarino** [comment]
I’d put this on [the homepage](https://github.com/everypolitician/everypolitician/issues/281)!

"Here’s what EveryPolitician is, here’s the data we hold, and here’s where it’s already being used."

**Comment by MyfanwyNixon** [comment]
@zarino The examples Tony gave above are now on https://www.mysociety.org/democracy/everypolitician/ - I'm sure they could be streamlined for homepage use.
